src/quo2/components/loaders/skeleton_list/view.cljs

```diff
@@ -0,0 +1,106 @@
+            [react-native.masked-view :as masked-view]
+            [react-native.reanimated :as reanimated]
+            [quo2.components.loaders.skeleton-list.style :as style]
+            [reagent.core :as reagent]
+            [quo2.components.loaders.skeleton-list.constants :as constants]))
+
+(defn skeleton-label
+  [animated? index parent-index]
+  (str (if animated? "skeleton-animated-" "skeleton-static-")
+       parent-index
+       "-"
+       index))
+
+(defn static-skeleton-view
+  [{:keys [index parent-index content color]}]
+  [rn/view
+   {:style               (style/container content)
+    :accessibility-label (skeleton-label false index parent-index)}
+   [rn/view {:style (style/avatar color)}]
+   [rn/view {:style style/content-container}
+    [rn/view
+     {:style (style/content-view
+              {:type    (if (= content :list-items) :message :author)
+               :index   index
+               :content content
+               :color   color})}]
+    [rn/view
+     {:style (style/content-view
+              {:type    (if (= content :list-items) :author :message)
+               :index   index
+               :content content
+               :color   color})}]]])
+
+(defn animated-skeleton-view
+  [{:keys [index parent-index content animated? style theme blur?] :as data}]
+  (let [loading-color  (colors/theme-colors colors/neutral-10 colors/neutral-60)
+        skeleton-color (cond
+                         blur?           colors/white-opa-5
+                         (= theme :dark) colors/neutral-90
+                         :else           colors/neutral-5)]
+    [masked-view/masked-view
+     {:style               {:height (get-in constants/layout-dimensions [content :height])}
+      :accessibility-label (skeleton-label true index parent-index)
+      :maskElement         (reagent/as-element (static-skeleton-view data))}
+
+     [rn/view
+      {:style {:flex             1
+               :background-color skeleton-color}}
+      (when animated?
+        [reanimated/linear-gradient
+         {:colors [skeleton-color skeleton-color loading-color skeleton-color skeleton-color]
+          :start  {:x 0 :y 0}
+          :end    {:x 1 :y 0}
+          :style  style}])]]))
+
+(defn- f-internal-view
+  [{:keys [content theme blur? parent-height animated?] :as props}]
+  (let [{window-width :width}   (rn/get-window)
+        translate-x             (reanimated/use-shared-value (- window-width))
+        animated-gradient-style (reanimated/apply-animations-to-style
+                                 {:transform [{:translateX translate-x}]}
+                                 {:width  window-width
+                                  :height "100%"})
+        skeleton-height         (get-in constants/layout-dimensions [content :height])
+        number-of-skeletons     (int (Math/ceil (/ parent-height skeleton-height)))
+        color                   (cond
+                                  blur?           colors/white-opa-5
```

Comment:
hmm I wonder should we make these checks with `blur?` also dependent on the correct theme? 🤔 
I understand that there is only a blur for dark here but still it might be safer to check that as it is clearer in the code of what it is, i.e it's `dark` theme and `blur?` true

Reply:
With the current changes, `blur ?` has no effect on the light theme. We can adapt it on next review if needed